developer_tools/cargo/cargo.py

```diff
@@ -154,7 +154,10 @@ def main():
     print("\n\nNOW PRINTING KEY RESULTS\n")
 
     for key in cargo_outdated_output:
-        if key == "Name" or key == "----":
+        if key == "Name":
```

Comment:
Try to follow the hint that was in the original lint:
```
cargo/cargo.py:157: [R1714(consider-using-in), main] Consider merging these comparisons with "in" to "key in ('Name', '----')"
```
The lint message suggests the preferred alternative.